app/models/content/content_item.rb

```diff
@@ -0,0 +1,56 @@
+module Content
+  class ContentItem < ApplicationRecord
```

Comment:
I'd considered doing that. I don't mind either way. I guess it just gets a bit weird if you refer within the module to just `Item`s